tensorflow/python/data/experimental/ops/as_dataframe.py

```diff
@@ -0,0 +1,27 @@
+# coding=utf-8
+# Copyright 2020 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Copied over from https://github.com/tensorflow/datasets/blob/master/tensorflow_datasets/core/as_dataframe.py - all code belongs to original authors
+
+import tensorflow as tf
+import numpy as np
+import dataclasses
+
+from tensorflow_datasets.core import dataset_info
```

Comment:
TensorFlow doesn't currently have any dependency on tensorflow_datasets, and it should stay that way to avoid a circular dependency. Instead of importing from tensorflow_datasets, we can port just the parts that are needed, and convert them to TensorFlow style (which doesn't currently support type annotations)